src/libraries/Microsoft.Extensions.Diagnostics.Abstractions/src/Metrics/IMeterFactory.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Diagnostics.Metrics;
 
-namespace Microsoft.Extensions.Metrics
+namespace Microsoft.Extensions.Diagnostics.Metrics
```

Comment:
Do API review folks need a heads-up on this? I don't know what the process is here, but for small changes like this in ASP.NET Core we do it over email.

Reply:
Yes, I am going to communicate that on the email.